src/opencl_bf_std_plug.c

```diff
@@ -247,21 +247,20 @@ void BF_select_device(struct fmt_main *fmt) {
 	HANDLE_CLERROR(clSetKernelArg(krnl[gpu_id], 6, sizeof(cl_mem), &buffers[gpu_id].S_box_gpu), "Set Kernel Arg FAILED arg6") ;
 
 	if (global_work_size) {
-		global_work_size =
-			global_work_size / local_work_size * local_work_size;
-		if (global_work_size > BF_N)
-			global_work_size = BF_N;
+		global_work_size = MIN(global_work_size / local_work_size * local_work_size, BF_N);
 		fmt->params.max_keys_per_crypt = global_work_size;
 	} else
 		find_best_gws(fmt);
 
-	if ((!self_test_running && options.verbosity >= VERB_DEFAULT) ||
```

Comment:
`options.verbosity >= VERB_DEFAULT` happens in `opencl_sl3_fmt_plug.c` too.

But there is no such condition in `opencl_autotune.h`, so I guess removal is intended.

Though conditions in `opencl_autotune.h` are much more complex. I guess they are applicable to formats with built-in mask. So I guess they are not needed here.

I tested. It seems ok.

Reply:
That verbosity check was back when we often muted LWS/GWS figures. As we don't do so anymore, it can be phased out from all formats.